### ISSUE TITLE
Doing `require 'backports'` is deprecated 

### ISSUE THREAD

**Issue by pjungermann** [jordansissel/fpm#1716]
With backport versions > 3.15.0 like 3.18.1 (didn't check others), you get the following warning.

```
Doing `require 'backports'` is deprecated and will not load any backport in the next major release.
Require just the needed backports instead, or 'backports/latest'.
```

**Comment by webketje** [comment]
On Linux Mint Cinnamon 19 (or other Debian/Ubuntu), the following error msg for building RPM/ Arch:
```
Doing `require 'backports'` is deprecated and will not load any backport in the next major release.
Require just the needed backports instead, or 'backports/latest'.
Process failed: /bin/bash failed (exit code 127). Full command was:["/bin/bash", "-c", "LANG=C
bsdtar -czf .MTREE --format=mtree --options='!all,use-set,type,uid,gid,mode,time,size,md5,sha256,link' .PKGINFO usr .INSTALL"] {:level=>:error}
Need executable 'rpmbuild' to convert dir to rpm {:level=>:error}
```
can be solved by manually installing: 

```
sudo apt install rpm libarchive-tools

**Comment by jordansissel** [comment]
This was fixed by #1727 and first released in fpm 1.12.0 (2021/01/19) :)
